### ISSUE TITLE
Ignore selected packages in the index file

### ISSUE THREAD

**Issue by belka-ew** [PackageKit/PackageKit#24]
We should have also an option of "origrestore" in the index field (or ignore since the package from the index file would be just ignored and a slackware package would be used instead of it). This would permit to reinstall the original package supplied from Slackware in case of an upgrade. Since it is something as local, for a single repository, obsolete package, "obsolete" field can be used for this.

**Comment by saxa** [comment]
Ok, thinking about it, and I'm not sure, but what happens if a package is not listed in the index file ?
I mean, if we have package-version in the system and then it simply is not in the index file, will Katja remove that package ?

**Comment by belka-ew** [comment]
no, otherwise it would remove everything, like SBo packages and so on....
